entry_types/scrolled/app/views/pageflow_scrolled/entries/show.html.erb

```diff
@@ -18,5 +18,7 @@
     <%= scrolled_entry_json_seed_script_tag(@entry, @seed_options || {}) %>
 
     <%= javascript_pack_tag 'pageflow-scrolled-frontend' %>
+    <%= public_i18n_javascript_tag(@entry) %>
+
```

Comment:
This includes all paged public translations. Let's keep scrolled translations in a separate scope (e.g. `pageflow_scrolled.public.*`) and extend the helper make the prefix configurable.